src/components/Comments/ConnectorWrapper.js

```diff
@@ -3,9 +3,9 @@ import {View, StyleSheet} from 'react-native';
 
 import {COLORS} from '../../utils/theme';
 
-const ConnectorWrapper = ({children, index}) => (
+const ConnectorWrapper = ({children, index, level}) => (
   <View style={styles.wrapper}>
-    <View style={styles.connector(index)} />
+    <View style={styles.connector(index, level)} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `level` is being used in the component but there's no prop validation for it. It's a good practice to validate props, especially when they are used in calculations or conditionals as it helps prevent unexpected behavior due to incorrect or missing values.
```diff
- const ConnectorWrapper = ({children, index, level}) => (
+ const ConnectorWrapper = ({children, index, level = 1}) => (
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->